Arelle/Arelle
# NL FR-NL 2.06 validation

#### Reason for change
FR-NL-2.06: A CDATA end sequence ("]]>") MAY NOT be used. 
A CDATA section, specifically the end sequence, will cause the SOAP processing to fail since the instance document is itself wrapped in a CDATA section.

The original wording of the rule stipulates that a CDATA "section" not be included, but prohibiting the CDATA end sequence specifically is a more accurate enforcement of the rule's intent.

#### Description of change
lxml parsing strips CDATA tags by default. Rather than make significant changes to parsing, we can load the raw file line-by-line checking against regex.

#### Steps to Test

Confirm [FR-NL-2.06] does not fire in "valid" examples.
Confirm [FR-NL-2.06] fires 4x running on `tests/resources/validation/NL/fr_nl/2.06/fail/invalid.xbrl`
Confirm [FR-NL-2.06] fires 4x running on `tests/resources/validation/NL/fr_nl/2.06/fail/invalid.zip/invalid.xbrl`

**review**:
@Arelle/arelle
